### DISCUSSION TITLE
is it possible to start with a pre-defined chat starter prompt

### DISCUSSION THREAD

**Discussion by resolver101757** [langflow-ai/langflow#153] [General]
I can see you can start with a template, fewshot and zeroshot prompts which as i understand it requires someone to type into the chat window to start the flow.  I want to langflow to start the flow by a pre-defined chat to get the end user started.  For example to start the chat off I want to say "you are a creative writer and the name of character 1 is ?" and then it goes onto the next flow.  

Is there currently a way to do this?

**Comment by ogabrielluiz** [comment]
I'm not sure I follow, but improvements to prompts are what we will focus on the most in the following weeks.

Your case seems similar to what we did in the SeriesCharacterChain. 
To have this kind of prompt working we would need support for custom prompts that have layers of input variables.

For example: 
```python
"""
you are a creative writer and the name of character 1 is {character}. 

{{input}}
Story: 
"""
```

And then you could say something like "Now write your story."
